### ISSUE TITLE
[Flaky Test] LegacyAppeal#location_history returns array of date, to_whom, by_whom

### ISSUE THREAD

**Issue by tomas-nava** [department-of-veterans-affairs/caseflow#13678]
## Background/context/resources
```ruby
LegacyAppeal#location_history returns array of date, to_whom, by_whom
     Failure/Error:
       expect(subject).to eq([
                               [today, first_location, "DSUSER"],
                               [today, second_location, "DSUSER"],
                               [today, third_location, "DSUSER"]
                             ])

       expected: [[Wed, 11 Mar 2020, "96", "DSUSER"], [Wed, 11 Mar 2020, "50", "DSUSER"], [Wed, 11 Mar 2020, "81", "DSUSER"]]
            got: [[Thu, 12 Mar 2020, "96", "DSUSER"], [Thu, 12 Mar 2020, "50", "DSUSER"], [Thu, 12 Mar 2020, "81", "DSUSER"]]

       (compared using ==)

       Diff:
       @@ -1,4 +1,4 @@
       -[[Wed, 11 Mar 2020, "96", "DSUSER"],
       - [Wed, 11 Mar 2020, "50", "DSUSER"],
       - [Wed, 11 Mar 2020, "81", "DSUSER"]]
       +[[Thu, 12 Mar 2020, "96", "DSUSER"],
       + [Thu, 12 Mar 2020, "50", "DSUSER"],
       + [Thu, 12 Mar 2020, "81", "DSUSER"]]
     # ./spec/models/legacy_appeal_spec.rb:680:in `block (3 levels) in <top (required)>'
```

 - [Circle CI Error](https://app.circleci.com/pipelines/github/department-of-veterans-affairs/caseflow/28617/workflows/745c6186-cb11-4354-b67e-610a492f7f8c/jobs/105807/parallel-runs/2?filterBy=FAILED)
 - Has the test already been skipped in the code?
   - [ ] Skipped
   - [x] Not Skipped

## Approach

Time box this investigation and fix.

Remember that if a test has been skipped for a decent amount of time, it may no longer map to the exact code.

If you reach the end of your time box and don't feel like the solution is in sight:
  - [ ] document the work you've done, including dead ends and research
  - [ ] skip the test in the code
  - [ ] file a follow on ticket
  - [ ] close this issue

**Comment by pkarman** [comment]
I thought this was fixed already via 9300909d33c0be3569fe74286ba924bba823194d

**Comment by tomas-nava** [comment]
It looks like that change wasn't merged into the branch when this test flaked; closing now.
